deposit/src/main/java/edu/unc/lib/deposit/normalize/DirectoryToBagJob.java

```diff
@@ -93,15 +94,21 @@ public void runJob() {
 
 		// Turn the bag itself into the top level folder for this deposit
 		PID containerPID = new PID("uuid:" + UUID.randomUUID());
-		com.hp.hpl.jena.rdf.model.Bag bagFolder = model.createBag(containerPID.getURI());
+		Bag bagFolder = model.createBag(containerPID.getURI());
 		model.add(bagFolder, labelProp, status.get(DepositField.fileName.name()));
 		model.add(bagFolder, hasModelProp, containerResource);
 		top.add(bagFolder);
 		
+		// Cache the source bag folder
+		pathToFolderBagCache.put(sourceFile.getName(), bagFolder);
```

Comment:
Can you get around directly referencing the cache in subclasses of AbstractFileServerToBagJob? It would be nice if subclasses didn't need to know about the implementation.